ldap/servers/slapd/back-ldbm/vlv_srch.c

```diff
@@ -539,20 +537,22 @@ vlvIndex_init(struct vlvIndex *p, backend *be, struct vlvSearch *pSearch, const
     }
 
     /* Create an index filename for the search */
-    if (vlvIndex_createfilename(p, &filename)) {
-        p->vlv_filename = slapi_ch_smprintf("%s%s%s", file_prefix, filename, file_suffix);
+    filename = vlvIndex_build_filename(p->vlv_name);
+    if (filename) {
+        p->vlv_filename = slapi_ch_smprintf("%s%s", filename, file_suffix);
 
         /* Create an attrinfo structure */
-        p->vlv_attrinfo->ai_type = slapi_ch_smprintf("%s%s", file_prefix, filename);
+        p->vlv_attrinfo->ai_type = filename;
         p->vlv_attrinfo->ai_indexmask = INDEX_VLV;
 
         /* Check if the index file actually exists */
         if (li != NULL) {
             vlvIndex_checkforindex(p, be);
         }
         slapi_timespec_expire_at(60, &(p->vlv_nextcheck));
+    } else {
+        slapi_log_err(SLAPI_LOG_ERR, "vlvIndex_init - Couldn't generate valid filename from Virtual List View Index Name (%s).  Need some alphabetical characters.\n", p->vlv_name, 0, 0);
```

Comment:
could p->vlv_name be NULL at this point ?

Reply:
This logging line is incorrect.  "vivIndex_init" should be its own parameter

Reply:
And you don't need the "0, 0" at the end.  That's an artifact from the old logging (not your fault, assuming copy/paste issue)

Reply:
Fixed